### DISCUSSION TITLE
Desoldering the 3,3V voltage regulator from the Arduino board

### DISCUSSION THREAD

**Discussion by Ruggerio150** [sanni/cartreader#127] [Need help]
![FE49C00B-4E05-4FAD-A91C-47AD40EFA100](https://user-images.githubusercontent.com/82999194/115687097-22227600-a35a-11eb-90e6-5604e9fa1351.png)

hello, which is the right regulator that i need to unsolder? In the instructions it says that it is next to the fuse for usb protection. but on the amazon site, the 3.3 volt regulator is elsewhere. much further to the right. have attached the picture here.

many thanks

**Comment by sanni** [comment]
You need to desolder the 5V regulator like shown on the instructions.
